src/main/java/org/isf/stat/manager/JasperReportsManager.java

```diff
@@ -522,6 +532,9 @@ public void getGenericReportPharmaceuticalStockCardExcel(String jasperFileName,
     public JasperReportResultDto getGenericReportPharmaceuticalStockWardPdf(Date date, String jasperFileName, Ward ward) throws OHServiceException {
     	
     	try{
+    		HashMap<String, Object> parameters = getHospitalParameters();
+            addBundleParameter(jasperFileName, parameters);
```

Comment:
indentation again; probably the whole file needs to be reformatted

Reply:
As for now I formatted only the blocks indicated. Thanks!